src/views/MainHome.tsx

```diff
@@ -73,6 +73,13 @@ export default class MainHome extends Component<MainHomeProps, MainHomeState> {
             this.setState({ roomInvite: invite });
         });
 
+        socket.on(Constants.LEAVE_ROOM, (data: { roomid: string }) => {
+            this.props.removeRoom(data.roomid);
+            if (this.state.currentRoomId === data.roomid) {
+                this.setState({ currentRoomId: '' });
```

Comment:
What does the user see after `currentRoomId` is set to empty string? Do they go back to the home page that the user sees after clicking `continue as guest`?

Reply:
Yeaa, that's the screen. What do you think about that interaction in terms of a ux perspective?

Reply:
I think that's fine. In a perfect world we could maybe have some notification but for now i think its fine